main.py

```diff
@@ -1,4 +1,6 @@
 import nltk
+# nltk.download('punkt')
```

Comment:
I had trouble running the chatbot backend app locally because of this..

@hajimss please commend.